include/scope.hpp

```diff
@@ -259,14 +258,14 @@ template <typename R, typename D> class unique_resource {
 
 public: // should be private
   template <typename RR, typename DD>
-    requires std::is_constructible_v<detail::box<R>, RR,
-                                     detail::_empty_scope_exit> &&
-                 std::is_constructible_v<detail::box<D>, DD,
-                                         detail::_empty_scope_exit>
-  unique_resource(RR &&r, DD &&d, bool should_run) noexcept(
-      noexcept(detail::box<R>(std::forward<RR>(r), detail::_empty_scope_exit{}))
-          && noexcept(detail::box<D>(std::forward<DD>(d),
-                                     detail::_empty_scope_exit{})))
+  requires std::is_constructible_v<detail::box<R>, RR,
+                                   detail::_empty_scope_exit> &&
+      std::is_constructible_v<detail::box<D>, DD, detail::_empty_scope_exit>
+      unique_resource(RR &&r, DD &&d, bool should_run)
```

Comment:
you might want to upgrade your llvm version (because clang format changed, and will block you in CI)

Reply:
Good catch